pkg/runner/grpc_test.go

```diff
@@ -294,6 +294,23 @@ func doGRPCTest(t *testing.T, l net.Listener, sec *atest.Secure, desc *atest.GRP
 				assert.Nil(t, err)
 			},
 		},
+		{
+			name: "test unary rpc not equal",
+			testCase: &atest.TestCase{
+				Request: atest.Request{
+					API:  unary,
+					Body: "{}",
+				},
+				Expect: atest.Response{
+					Body: getJSONOrCache("unaryneq", &testsrv.HelloReply{
```

Comment:
Actually, I didn't quite understand why using `getJSONOrCache`. I guess you only want to convert a struct into JSON string. A cached function might return an unpredictable result.

Reply:
ok i will do a bypass here